### ISSUE TITLE
User executors release broke core executors autocompleting

### ISSUE THREAD

**Issue by ivan-velasco** [Thiht/vscode-venom#11]
Issue: Autocomplete Not Working for User Executor in v1.7.0
v1.7.0 - https://github.com/Thiht/vscode-venom/pull/7

**Steps to Reproduce:**

1. Create a user executor and a `testsuite.venom.yml` file.
2. Add a test case with the type set to the new user executor.

**Expected Behavior:** Autocomplete should display the executor.
**Actual Behavior:** No autocomplete suggestions are available.

I attempted disabling and re-enabling the plugin. While it loads, it initially throws IntelliSense errors. However, these errors disappear when I type a new line.

**Comment by Thiht** [comment]
Hi @ivan-velasco,
Thanks for the report!
This should be fixed in the new 1.7.1 release.

**Comment by ivan-velasco** [comment]
@Thiht thanks for the update, I have tested it out and no luck. Can you give it a try?

**Comment by Thiht** [comment]
Maybe I didn't understand the issue correctly then, sorry!

This is what I fixed (notice `custom_executor_custom` in the completion, that wasn't here before)

![vscode-venom](https://github.com/user-attachments/assets/965620a6-ccdb-485e-9567-077cb84dce1e)

Can you give me more details on the issue?

**Comment by ivan-velasco** [comment]
Yes you understood correctly the issue I was facing. After applying the release, I don't get autocomplete to appear.. 

I do notice that the testsuites don't show the `description` for the `venom.schema.json` it only shows `venom.schema.json`

![vs-code-001](https://github.com/user-attachments/assets/74f993e6-2a84-415d-b6d0-c6b043d8908c)

**Comment by Thiht** [comment]
Does it work if you manually trigger the autocomplete? `^ + space` on Mac, or I think `ctrl + space` on Windows/Linux

**Comment by ivan-velasco** [comment]
Yup I tried this shortcut option also and no luck.
<img width="592" alt="Screenshot 2024-08-21 at 3 56 47 PM" src="https://github.com/user-attachments/assets/56b587df-0cb7-45f9-a40d-8dc778f3dcbb">

**Comment by Thiht** [comment]
That's weird, I don't know what's happening

I'll try to reproduce, could you give me these additional info:

- Your OS + version
- Your VSCode version
- The VSCode extensions you have installed (it would be great if you could try with just the vscode-venom and YAML extensions enabled, maybe there's a conflict with another extension)
  - If you happen to know what extension gives the completions show in your capture (`!And`, `!Base64`, etc.), maybe it's linked
- Just in case, if you could confirm you're using vscode-venom 1.7.1 🙏 
- Your venom version (should not be relevant, but who knows)
- In the "Output" panel of VSCode (`shift + command + u` on Mac) you can choose "Venom" in the combo box, is there anything relevant?
- In the developer console (Help menu -> Toggle Developer Tools -> Console tab), do you see some errors or warnings?
- If you can, could you send me a zip or a Git repo with the minimal files to reproduce the issue?

**Comment by ivan-velasco** [comment]
macOS Sonoma 14.5 (23F79)
VSCode: Version: 1.92.2
vscode-venom 1.7.1 
venom version: custom built binary with base being 1.2.0

> The VSCode extensions you have installed (it would be great if you could try with just the vscode-venom and YAML extensions enabled, maybe there's a conflict with another extension)

I disabled all but the two extensions mentioned, no change in behavior. 

>If you happen to know what extension gives the completions show in your capture (!And, !Base64, etc.), maybe it's linked

extension YAML 1.15.0

> In the "Output" panel of VSCode (shift + command + u on Mac) you can choose "Venom" in the combo box, is there anything relevant?

```
2024-08-22 10:16:37.877 [info] Loading venom.schema.json
2024-08-22 10:16:37.877 [info] Registering test suite schema to YAML extension
2024-08-22 10:16:37.877 [info] Loading venom-custom-executor.schema.json
2024-08-22 10:16:38.003 [info] Registering custom executor schema to YAML extension
2024-08-22 10:16:38.049 [info] Computed Venom lib directory: /Users/ivanvelasco/Code/**REDACTED**/lib
2024-08-22 10:16:38.327 [info] Computed Venom lib directory: /Users/ivanvelasco/Code/**REDACTED**/lib
2024-08-22 10:16:38.462 [info] Loading custom executor /Users/ivanvelasco/Code/**REDACTED**/lib/executors/get_past_future_date_time.yml
```

> In the developer console (Help menu -> Toggle Developer Tools -> Console tab), do you see some errors or warnings?

```
INFO Started local extension host with pid 39216.
log.ts:439   ERR No registered selector for ID: ms-vscode.npm-command
console.ts:137 [Extension Host] filesize is active
console.ts:137 [Extension Host] [vscode-cpptools-api] C/C++ extension is not installed
```

I then installed the extension and this error got removed.

> If you can, could you send me a zip or a Git repo with the minimal files to reproduce the issue?

I will upload this bit later today

**Comment by ivan-velasco** [comment]
@Thiht I found the root cause for the issue 

`yaml` extension has a `completion` setting that was take precedence over vscode-venom autocomplete. I disabled and venom extension started working.
<img width="272" alt="Screenshot 2024-08-30 at 2 13 11 PM" src="https://github.com/user-attachments/assets/e8d2a7b6-05ae-4e3a-86ca-3ea852a4922b">

I did however identify an issue with built in executors not appearing in the autocomplete list, besides `exec` executor. I will log a separate issue.
